packages/account/src/Sections/Assessment/FinancialAssessment/financial-assessment.tsx

```diff
@@ -379,175 +386,171 @@ const FinancialAssessment = observer(() => {
-                                                    <SelectNative
-                                                        placeholder={localize('Please select')}
-                                                        name='employment_status'
-                                                        label={localize('Employment status')}
-                                                        list_items={getEmploymentStatusList()}
-                                                        value={values.employment_status}
-                                                        error={
-                                                            touched.employment_status
-                                                                ? errors.employment_status
-                                                                : undefined
-                                                        }
-                                                        onChange={e => {
-                                                            setFieldTouched('employment_status', true);
-                                                            handleChange(e);
-                                                        }}
-                                                    />
-                                                </MobileWrapper>
-                                            </fieldset>
-                                        )}
+                                </div>
+                            )}
+                            <FormBody scroll_offset={getScrollOffset()}>
+                                <FormSubHeader
+                                    title={localize('Financial information')}
+                                    subtitle={`(${localize('All fields are required')})`}
+                                />
+                                <FormBodySection
+                                    side_note={localize('We’re legally obliged to ask for your financial information.')}
+                                >
+                                    <fieldset className='account-form__fieldset'>
+                                        <DesktopWrapper>
+                                            <Dropdown
+                                                placeholder={localize('Source of income')}
+                                                is_align_text_left
+                                                name='income_source'
+                                                list={getIncomeSourceList()}
+                                                value={values.income_source}
+                                                onChange={handleChange}
+                                                handleBlur={handleBlur}
+                                                error={touched.income_source && errors.income_source}
+                                            />
+                                        </DesktopWrapper>
```

Comment:
Let’s replace DesktopWrapper and MobileWrapper with just is_desktop and is_mobile.
how do you think?

Reply:
For now I'll keep this as it is not related to the card. There are so many ways to render responsive 😆 now. May be team can decide which to stick with